certora/ArrangerConduit.spec

```diff
@@ -602,16 +601,18 @@ rule drawFunds(address asset, address destination, uint256 amount) {
     mathint balanceOfConduitAfter = gem.balanceOf(currentContract);
     mathint balanceOfDestinationAfter = gem.balanceOf(destination);
 
-    assert balanceOfConduitAfter == balanceOfConduitBefore - amount, "balance of conduit did not decrease by amount";
-    assert balanceOfDestinationAfter == balanceOfDestinationBefore + amount, "balance of destination did not increase by amount";
+    assert currentContract != destination => balanceOfConduitAfter == balanceOfConduitBefore - amount, "balance of conduit did not decrease by amount";
+    assert currentContract == destination => balanceOfConduitAfter == balanceOfConduitBefore, "balance of conduit changed";
+    assert currentContract != destination => balanceOfDestinationAfter == balanceOfDestinationBefore + amount, "balance of destination did not increase by amount";
+    assert currentContract == destination => balanceOfDestinationAfter == balanceOfDestinationBefore, "balance of destination changed";
```

Comment:
If `currentContract == destination`, `balanceOfConduit == balanceOfDestination`, then you just need to check it once.

Reply:
ok, done.